src/main/java/org/jenkinsci/plugins/fluentd/Fluentd.java

```diff
@@ -90,8 +90,15 @@ public void perform(@Nonnull Run<?, ?> build, @Nonnull FilePath workspace, @Nonn
     private boolean send(String json, String jsonFromFile, @Nonnull Run<?, ?> build, @Nonnull TaskListener listener) throws InterruptedException {
         final Map<String, String> envVars = getEnvVariables(build, listener);
 
+        FluentLogger logger = FluentLoggerHolder.getLogger();
+
+        if (logger == null) {
+            listener.error("Can't send data: fluentd logger is not initialised");
```

Comment:
Nit: "initialised" is British English, but we should use American English "initialized" (likewise below).